docs/content/docs/tasks_usage/metrics.md

```diff
@@ -5,22 +5,48 @@ type: docs
 aliases: ["metrics.md"]
 ---
 
-
-# Metrics
-Open Service Mesh (OSM) generates detailed metrics for all services communicating within the mesh. These metrics provide insights into the behavior of services in the mesh helping users to troubleshoot, maintain and analyze their applications.
+## Table of Contents
```

Comment:
The docs site generates and shows a table of contents for each doc on the right side of the page, so I'd be in favor of removing it here.